### DISCUSSION TITLE
Linear Regression/OLS single machine benchmark result

### DISCUSSION THREAD

**Discussion by LiangdeLI** [sgkit-dev/sgkit#792] [Show and tell]
Hi, @hammer @tomwhite @eric-czech @jeromekelleher , this is the plot of single machine benchmark result for OLS 1000 genome data(chr 1-22). 

[1kg_ols_single_wall_total.pdf](https://github.com/pystatgen/sgkit/files/7820553/1kg_ols_single_wall_total.pdf)

I created synthetic numeric phenotypes for those genotype data. It seems Sgkit performed well compared to Hail and PLINK has a bad scale in multithread OLS.

So by now the single machine experiments should have done and I'm now doing the multi-nodes distributed experiments for Sgkit(Dask) and Hail(Spark), reading data from HDFS.

The meeting time clashes with my class this term (next 10 weeks), so I probably cannot attend the meeting but will keep posting results, asking questions on GitHub.

**Comment by jeromekelleher** [comment]
Very nice @LiangdeLI, thanks! Do you have a link to the code? (simpler than asking lots of questions!)
